CITATION.bib

```diff
@@ -18,6 +18,6 @@ @ARTICLE{2020SciPy-NMeth
   year    = {2020},
   volume  = {17},
   pages   = {261--272},
-  adsurl  = {https://rdcu.be/b08Wh},
+  adsurl  = {https://ui.adsabs.harvard.edu/abs/2020NatMe..17..261V},
```

Comment:
@tupui what do you think about the alternative - just making this a regular URL? I'm a fan of astrophysics, but I don't think this should be domain specific.

Reply:
There is the `url` keyword for the generic URL. The `adsurl` keyword is, by identity, for an ADS link.

Reply:
I went ahead and added this field (`url`) as well since it was previously not present.

Reply:
Mmm I would remove this field entirely. The DOI is the real reference URL and we should not have to rely on any external/domain specific service.